modules/oe_content_project/oe_content_project.install

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * The OpenEuropa Content Project installation file.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_install().
+ */
+function oe_content_project_install() {
+  // Add 'EU Programme' vocabulary.
+  if (!\Drupal::service('config.installer')->isSyncing()) {
+    \Drupal::service('rdf_skos.skos_graph_configurator')->addGraphs([
+      'eu-programme' => 'http://publications.europa.eu/resource/authority/eu-programme',
+    ]);
+  }
```

Comment:
In these cases we usually exit early rather than wrapping code in ifs, as it improves readability, so something like:
```php
  if (\Drupal::service('config.installer')->isSyncing()) {
    return;
  }

  // Add 'EU Programme' vocabulary.
  \Drupal::service('rdf_skos.skos_graph_configurator')->addGraphs([
    'eu-programme' => 'http://publications.europa.eu/resource/authority/eu-programme',
  ]);
```

Reply:
Changed